packages/node_modules/@node-red/editor-client/src/js/ui/palette-editor.js

```diff
@@ -385,54 +385,62 @@ RED.palette.editor = (function() {
     var catalogueCount;
     var catalogueLoadStatus = [];
     var catalogueLoadStart;
-    var catalogueLoadErrors = false;
 
     var activeSort = sortModulesRelevance;
 
-    function handleCatalogResponse(err,catalog,index,v) {
-        const url = catalog.url
-        catalogueLoadStatus.push(err||v);
-        if (!err) {
-            if (v.modules) {
-                v.modules = v.modules.filter(function(m) {
-                    if (RED.utils.checkModuleAllowed(m.id,m.version,installAllowList,installDenyList)) {
-                        loadedIndex[m.id] = m;
-                        m.index = [m.id];
-                        if (m.keywords) {
-                            m.index = m.index.concat(m.keywords);
-                        }
-                        if (m.types) {
-                            m.index = m.index.concat(m.types);
-                        }
-                        if (m.updated_at) {
-                            m.timestamp = new Date(m.updated_at).getTime();
-                        } else {
-                            m.timestamp = 0;
-                        }
-                        m.index = m.index.join(",").toLowerCase();
-                        m.catalog = catalog;
-                        m.catalogIndex = index;
-                        return true;
+    function refreshCatalogues (done) {
+        catalogueLoadStatus = [];
+        catalogueCount = catalogues.length;
+        loadedList = []
+        loadedIndex = {}
+        loadedCatalogs.length = 0
+        let handled = 0
+        for (let index = 0; index < catalogues.length; index++) {
+            const url = catalogues[index];
+            $.getJSON(url, {_: new Date().getTime()},function(v) {
+                loadedCatalogs.push({ index: index, url: url, name: v.name, updated_at: v.updated_at, modules_count: (v.modules || []).length })
+                handleCatalogResponse({ url: url, name: v.name},index,v);
+            }).fail(function(jqxhr, textStatus, error) {
+                console.warn("Error loading catalog",url,":",error);
```

Comment:
```suggestion
                $(".red-ui-palette-module-shade-status").text(RED._('palette.editor.errors.catalogLoadFailed'));
                console.warn("Error loading catalog",url,":",error);
```

Reply:
Not useful. I don't know if this case happens often.
We'll leave it like that for now.